provider/blob/src/blob_schema.rs

```diff
@@ -125,6 +125,12 @@ impl<'a> ZeroMapKV<'a> for Index32U8 {
     type OwnedType = Box<Index32U8>;
 }
 
+#[inline]
+const fn indexify(s: &[u8]) -> &Index32U8 {
```

Comment:
nit: probably include a comment explaining why this is always safe and why we're using a different function here

Reply:
I don't know why though :D

Reply:
it's always safe because `Index32U8` is just a transparent wrapper around `[u8]`, as generated by `make_varule`.

A different function? idk you're the one who made the change; presumably because it's easier to document